ftl/BUILD

```diff
@@ -0,0 +1,24 @@
+package(default_visibility = ["//visibility:public"])
+
+load(
+    "@io_bazel_rules_docker//docker/contrib/python:image.bzl",
+    "py_image",
+)
+load(
+    "@io_bazel_rules_docker//docker:docker.bzl",
+    "docker_push",
+    "docker_build",
+)
+load("@subpar//:subpar.bzl", "par_binary")
+
+py_library(
+    name = "ftl_lib",
+    srcs = glob([
+        "__init__.py",
+        "common/*.py",
```

Comment:
I think we want a `py_library` and corresponding `py_test` for each of the logic elements.  Not sure if you saw my other comment about adding unit tests?

Reply:
Yeah, working on tests now. I was planning on adding tests for everything that's not abstract, make sense?

As far as fakes, I'm not entirely sure what you had in mind. Something like a MemoryCache and a MemoryContext? I'm not sure what a fake builder would look like.

Reply:
Yeah, I think creating fakes on demand is probably fine.  IDK that a fake builder makes sense either.  Tests for everything instantiable SGTM.